frontend-react/src/components/Admin/AdminReceiverDashboard.tsx

```diff
@@ -529,63 +678,165 @@ function ModalInfoRender(props: { subData: AdmConnStatusDataType[] }) {
 
     return (
         <GridContainer className={"rs-admindash-modal"}>
-            <Grid className={"modal-info-title"}>
-                Results for connection verification check
-            </Grid>
-            <Grid row className={"modal-info-row"}>
-                <Grid className={"modal-info-label"}>Org:</Grid>
-                <Grid className={"modal-info-value"}>
-                    {dataItem.organizationName} (id: {dataItem.organizationId})
-                </Grid>
-            </Grid>
-
-            <Grid row className={"modal-info-row"}>
-                <Grid className={"modal-info-label "}>Receiver:</Grid>
-                <Grid className={"modal-info-value"}>
-                    {dataItem.receiverName} (id: {dataItem.receiverId})
-                </Grid>
-            </Grid>
-
-            <Grid row className={"modal-info-row"}>
-                <Grid className={"modal-info-label"}>Result:</Grid>
+            {props.subData.map((dataItem) => (
```

Comment:
[?] are these changes just nesting the existing content under the map? Is there a way to test this behavoir for multiple data points

Reply:
Yeah. Each "slice of day" could contain multiple results. Right now it's every two hours so it's pretty much 1:1... but if you change the slices to every 6 hrs (so there are 4x-slices-per-day), the there would be multiple results for that slice.

In that situation, the slice becomes "candy cane" striped. 

I'll add another comment at that loop.